pipelines/data_embedding_pipeline/data_embedding_pipeline.py

```diff
@@ -16,5 +16,5 @@ def data_embedding_pipeline() -> None:
     """
     mind_df, nhs_df = fetch_data()
 
-    mind_df = embed_data(mind_df)
-    nhs_df = embed_data(nhs_df)
+    mind_df = embed_data(mind_df, "base", "v1.0", "mind_data")
```

Comment:
Looks like the `embed_data()` step is not meant to return anything from the `None` type hint.